src/gl/interface/gl-pixels.ads

```diff
@@ -345,6 +346,9 @@ private
    for Data_Format use (Stencil_Index   => 16#1901#,
                         Depth_Component => 16#1902#,
                         Red             => 16#1903#,
+                        Green           => 16#1904#,
+                        Blue            => 16#1905#,
+                        Alpha           => 16#1906#,
```

Comment:
What are those doing here? This enum is for the `glTexImage?D` functions. According to the [spec](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glTexImage2D.xhtml), they don't take `GL_GREEN`, `GL_BLUE` or `GL_ALPHA`.

Reply:
OK.
I'll remove them.